modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIEndpointFields.tsx

```diff
@@ -33,6 +34,7 @@ export default function BaseAPIEndpointFields({
 	apiApplicationBaseURL,
 	basePath,
 	data,
+	disabledProperties,
```

Comment:
As the value could be undefined, we should set the default value to false here
`isEdition = false`. So in cases like the `CreateAPIEndpoint` this would be false, and in `EditAPIEndpoint` it would be true